README.md

```diff
@@ -19,15 +19,34 @@ pip install log-enricher
 configuration
 -------------
 
-The log-enricher is configured with a simple dictionary:
-```
+Log-enricher runs Enrichers, and is configured with a simple dictionary:
+
+```python
+from log_enricher import initialize_logging
+
+from log_enricher.enrichers import Enricher
+
+class RequestIDEnricher(Enricher):
+    def __call__(self) -> Dict[str, Any]:
+        return {"request_id": get_request_id()}
+
 config = {
     "handlers": "plain"/"structured",
     "log_level": "DEBUG",
     "app_version": "xyz",
     "release_stage": "staging",
     "loggers": ["py", "mylogger"]
 }
+
+initialize_logging(config, enrichers=[RequestIDEnricher()])
 ```
+
 Logs will be output in a plain, console-friendly format if `handlers` is 
-`"plain"`, or in a structured JSON format if it is `"structured"`.
\ No newline at end of file
+`"plain"`, or in a structured JSON format if it is `"structured"`.
+
+`initialize_logging()` will talk to and configure the `logging` subsystem. Once that has been called, `logging`
+will use the new configuration.
+
+# TODO: HOW DO ENRICHERS WORK?
```

Comment:
Any method returning a dict can be used to enrich log records - https://github.com/arni-inaba/log-enricher/blob/master/log_enricher/enrichers.py#L10-L13

Reply:
The key-value pairs in the dict are added as attribute-value pairs to the log record.

Reply:
Roger! And the methods that those records call, there is no changing log context - it is just assumed that those methods should work at any time the logging system logs something?

Reply:
Yup! It's the responsibility of the method to return something sensible when called.